kube-core/src/response.rs

```diff
@@ -69,12 +114,12 @@ pub struct StatusDetails {
     ///
     /// Some errors may indicate the client must take an alternate action -
     /// for those errors this field may indicate how long to wait before taking the alternate action.
-    #[serde(default, skip_serializing_if = "num::Zero::is_zero")]
+    #[serde(default, skip_serializing_if = "is_u32_zero")]
```

Comment:
I don't understand this, did `num::Zero` not work?

Reply:
https://github.com/kube-rs/kube-rs/blob/master/kube-core/Cargo.toml#L25
kube-core does not depend on `num` (and I decided not to add a new dependency for two trivial functions).

I guess compile error didn't appear earlier because `derive(Serialize)` was not enabled so this attribute was silently ignored.